common/db_test.go

```diff
@@ -656,6 +659,83 @@ func TestLoadWinningTickets_GivenTwoSessionsWithTickets_ReturnsAllTickets(t *tes
 	assert.Equal(recipientRand1, recipientRands[1])
 }
 
+func TestInsertMiniHeader_ReturnsFindLatestMiniHeader(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	assert := assert.New(t)
+	require := require.New(t)
+	require.Nil(err)
+
+	h0 := defaultMiniHeader(t)
+	err = dbh.InsertMiniHeader(h0)
+	require.Nil(err)
+
+	h0db, err := dbh.FindLatestMiniHeader()
+	require.Nil(err)
+	assert.Equal(*h0, *h0db)
```

Comment:
Just a heads up - [assert.Equal()](https://godoc.org/github.com/stretchr/testify/assert#Equal) will compare the referenced values when comparing 2 pointers so we don't need to explicitly deference pointers in this test file.

Reply:
Cool, I thought so but wasn't entirely sure!

Reply:
Fixed in cb0ca5e